### PR TITLE
Modify api-request:fetchWorkflows, normalize recorded requests

### PR DESCRIPTION
**Pull request by bantic**
Change api-request:fetchWorkflowIds to `fetchWorkflows` and return both id and name.

Also write normalization to use by pollyjs so that more of the headers that arbitrarily vary from request-to-request will stay the same. This will minimize the diff when re-recording requests.

[commit a39dcea22cf151c050cab08e315f43bec1657e2e](https://github.com/Addepar/github-metrics/pull/9/commits/a39dcea22cf151c050cab08e315f43bec1657e2e) is an example of the diff that now occurs when refreshing recordings. We cannot avoid the binary "text" body changing, but that is the only change now when re-recording requests.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,7 +40,6 @@
     "fetch-mock": "^9.11.0",
     "jest": "^26.6.3",
     "setup-polly-jest": "^0.9.1",
-    "string.prototype.replaceall": "^1.0.5",
```
Comment (bantic) [inline package.json:43]: Dropped this in favor of more explicit normalization steps on the HAR records

File: tests/setup-polly.ts [hunk 2]
```diff
@@ -1,18 +1,79 @@
...
 // @ts-ignore
 import { setupPolly } from 'setup-polly-jest';
-
 // @ts-ignore
 import FSPersister from '@pollyjs/persister-fs';
+import { Har, Header } from 'har-format';
+
+type HeaderDefaults = {
+  [key: string]: string;
+};
```
Comment (twokul) [inline tests/setup-polly.ts]: Can we use [`Record`](https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeystype) here?

```suggestion
type HeaderDefaults = Record<string, string>;
```

**Review by twokul — APPROVED** [review]
One small comment, lgtm otherwise

File: tests/setup-polly.ts (hunk 2 above)
Comment (bantic) [reply tests/setup-polly.ts]: 👍 done!

**Pull request merged**